test/models/cart_test.rb

```diff
@@ -14,4 +14,75 @@ class CartTest < ActiveSupport::TestCase
     assert_equal({ "1" => 2, "2" => 1}, cart.contents)
   end
 
+  test 'can minus_item from cart' do
+    cart = Cart.new({ "1" => 1})
+    cart.add_item(1)
+    cart.add_item(2)
+    cart.minus_item(2)
+
+    assert_equal({"1" =>2, "2"=> 0}, cart.contents)
+  end
+
+  test 'cart can give me the total sum of the items in cart' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.